index.html

```diff
@@ -103,7 +103,7 @@ <h4 class="d-flex justify-content-between align-items-center mb-3">
           </h4>
             <div class="mb-3">
               <label for="">Eagle CAD Layer</label>
-              <input type="text" class="form-control" id="eagleLayer" placeholder="Eagle CAD Layer" value="tPlace" required>
+              <input type="text" class="form-control" id="eagleLayer" placeholder="Eagle CAD Layer" value="200" required>
```

Comment:
Did we mean to change the default layer from tPlace to "200"?

Reply:
For the incremental system to work the layer needs to be a number, ( should probably add some verification code for this ) I chose layer 200 as this is the default bmp layer used by the bitmap import script that ships with Eagle

Reply:
That's a good point though - @mhenstell is that causing you issues?

Reply:
The issue I ran into is that this changes the default on the frontend:

![Screen Shot 2023-06-20 at 5 36 46 PM](https://github.com/gfwilliams/svgtoeagle/assets/359829/1e8860b5-f9f8-47e8-991d-d71ec5622259)

I just changed it to Dimension or whatever I was using, but this might confuse people. Or maybe not, I've never used the BMP script.

Reply:
The color splitting code requires layers to be used as numbers to increment the active layer dynamically.

The ideal solution would be to add functionality for the script to auto convert one into the other, though that's not dynamically possible with just Scripts in Eagle, (would be with ULP's)

Personally I'd opt for only allowing numbers as input for the layer field. Especially since it's not hard to find the layer ID within eagle.

We could change the default to 21 (tPlace), though Eagle auto enables a layer when the 'Change' command is invoked. So I don't see any issue with using 200

Reply:
I just tried on a fresh board and it does appear to work ok, creating the layer 200. I think maybe the best thing to do is to just add some docs to the right that mention common layers - I'll do that now

Reply:
Actually I take that back - on a fresh board it breaks completely with 'Invalid Layer 200'. I've changed the default number back to tPlace (but using 21 instead) and added some comments for common layer numbers.

While having multiple layers is cool, the whole point of this is for it to 'just work', so anything that adds friction isn't that helpful.

What about allowing commas in the CAD Layer? So you could just mention multiple layers like `Top,Bottom` and then having two colors of output would automatically swap between those? Seems easy to understand, can use the layer names, and is also more flexible?

Reply:
Then we should also detect the total amount of colours in the SVG and give some user feedback on that no?

we can also leverage the 'make new layer command' `LAYER layer_number layer_name` to make sure the layer exists, though this would overwrite the name of that layer if it existed before

Reply:
> Then we should also detect the total amount of colours in the SVG and give some user feedback on that no?

Yes - that would be a good way of doing it actually - if the layers thing is enabled, create a popup saying `Your SVG has # colours', but you have specified one layer. Enter more layer names/numbers separated by commas`.

Even better if it could show you the colours and their order, and then you could name the layers accordingly :)

Reply:
Aight I'll look into it, bit busy this week but I'll def work towards that